tests/src/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -80,6 +80,7 @@ class ApiGwTests
       }
       finally {
         val deleteresult = wsk.api.delete(basepathOrApiName = testbasepath)
+        Thread.sleep(5000)  // Test suite will overrun the local dev throttling limit
```

Comment:
@rabbah @markusthoemmes What you think about this approach to sleep, to avoid hitting the 60 api calls per minute limit in tests?